rust/main/agents/relayer/src/msg/metadata/base.rs

```diff
@@ -329,10 +330,13 @@ impl BaseMetadataBuilder {
     }
 
     pub async fn get_merkle_leaf_id_by_message_id(&self, message_id: H256) -> Result<Option<u32>> {
-        let merkle_leaf = self
+        // FIXME It's a stub for hyperlane-ton e2e test debugging.
+        // Remove when debugging is finished.
+        let _merkle_leaf = self
             .db
             .retrieve_merkle_leaf_index_by_message_id(&message_id)?;
-        Ok(merkle_leaf)
+        Ok(Some(1))
+        //Ok(merkle_leaf)
```

Comment:
I wonder if this change was intentionally committed?

Reply:
The PR is a work in progress so we will fix all issues and remove debug comments as soon as possible.